TOOLS/dylib-unhell.py

```diff
@@ -24,8 +24,8 @@ def is_user_lib(objfile, libname):
 
 def otool(objfile):
     command = "otool -L %s | grep -e '\t' | awk '{ print $1 }'" % objfile
-    output  = subprocess.check_output(command, shell = True)
-    return filter(partial(is_user_lib, objfile), output.split())
+    output  = subprocess.check_output(command, shell = True, universal_newlines=True)
+    return list(filter(partial(is_user_lib, objfile), output.split()))
```

Comment:
It looks like order isn't important? why not just create the set() here?

Reply:
Then, you wouldn't need to create a new set() at line 38, just assign the existing one.

Reply:
no reason, i just didn't think about optimising this part of the code. i mean that change would even be valid without having a need to fix it. i was just dumbly planning to make the code work consistently between the python versions, eg just return the same type as before/with python2.

i pushed the changed and hope that is what you had in mind.